ext/storage_sources/s3_store/README.md

```diff
@@ -0,0 +1,55 @@
+# S3 Store Extension
+
```

Comment:
We should mention that this is a WiredTiger extension and the guide/readme here is to build WiredTiger with the extension enabled. We are building WiredTiger, not just the extension alone.